_posts/2017-08-26-a11y-and-other-numeronyms.md

```diff
@@ -0,0 +1,70 @@
+
+For instance, "9-1-1" is synonymous with "help", "24/7" means "24 hours a day, 7 days a week", and Y2K is for "year 2000" or "the year 2000 problem." These are all examples of numeronyms.
+
+As illustrated by the previous numeronyms, sometimes they're used to replace entire words with numbers, or they use a combination of letters and numbers to combine words. In the case of a11y, [Adrian Roselli](https://twitter.com/aardrian) explains:  
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Tip: “a11y” is a numeronym for accessibility. Keep first &amp; last letter, between them put the count of letters removed — in this case eleven.</p>&mdash; Adrian Roselli 🗯 (@aardrian) <a href="https://twitter.com/aardrian/status/801084161368920064">November 22, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>  
+
+
+### Other numeronyms  
+
+The following is a short list of numeronyms you may have seen, or even used, but didn't know what they were called.  
+
+<dl>
+  <dt style="font-weight: bold;">i11l</dt>
+  <dd style="padding-left: 1em;">international</dd>
+  <dt style="font-weight: bold;">i18n</dt>
+  <dd style="padding-left: 1em;">internationalization</dd>
+  <dt style="font-weight: bold;">l10n</dt>
+  <dd style="padding-left: 1em;">localization</dd>
+  <dt style="font-weight: bold;">n7m</dt>
+  <dd style="padding-left: 1em;">Numeronym</dd>
+  <dt style="font-weight: bold;">v12n</dt>
+  <dd style="padding-left: 1em;">Virtualization</dd>
+  <dt style="font-weight: bold;">W3</dt>
+  <dd style="padding-left: 1em;">World Wide Web</dd>
+  <dt style="font-weight: bold;">W12t</dt>
+  <dd style="padding-left: 1em;">Web Development</dd>
+  <dt style="font-weight: bold;">W3C</dt>
+  <dd style="padding-left: 1em;">World Wide Web Consortium</dd>
+</dl>
+
+
+## Back to a11y  
+
+While numeronyms have been around for quite some time, they're not always immediately understandable. As a defining trait of accessibility is about being inclusive, one should keep the following in mind when using a11y:  
+
+1. If you can say or write "accessibility", do so! A11y (often pronounced "A-1-1-Y", "A-eleven-Y", and liberally as "ally") is meant as shorthand for "accessibility". So whenever possible, say "accessibility", even if written as a11y. The best place to use the numeronym are mediums like Twitter where space is limited and every character counts.   
+2. A numeronym will be easier to understand the more it is used in context with its true spelling. So if writing, or giving a presentation, be sure to make your audience aware a11y = accessibility as soon as it's introduced. Sure, people will likely be able to piece it together over time, but no reason not to make it immediately clear.  
```

Comment:
> a11y = accessibility
👇
> a11y == accessibility
😉

Reply:
i see what you did here, but I reference Adrian's article "a11y = Accessibility", and I'd like to keep that consistent. :)

Reply:
Lol, sounds good.